src/assets/Gas.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M9 8V8.47188V9V9.47188V14V15H2V14V5.5C2 6.32728 2.67272 7 3.5 7H7.5C8.32728 7 9 6.32728 9 5.5V8ZM15 4.82812V5.99677L14.7192 5.93369C14.32 5.84403 14 5.47051 14 5V3.41421L14.7085 4.12273C14.8955 4.30973 15 4.56144 15 4.82812ZM2 2C2 1.4476 2.4476 1 3 1H3.5C2.67434 1 2 1.66984 2 2.5V2ZM7.5 1H8C8.55135 1 9 1.44811 9 2V2.5C9 1.66984 8.32566 1 7.5 1Z" stroke="#9B9BA7" stroke-width="2"/>
+</svg>
```

Comment:
this renders broken for me

Reply:
need to turn off simplify stroke when exporting from figma